app/controllers/external_controller.rb

```diff
@@ -13,7 +13,7 @@ def create_user
       u.language = extract_language_code user_info['locale']
     end
 
-    session[:user_id] = user.id
+    session[:session_token] = user.session_token
```

Comment:
I think for `openid_connect` this is the endpoint for both signup/signin.
So I believe this will fail to authenticate users because it's not generating/renewing session tokens when they authenticate?

Reply:
Yeah this just needs a generate token before saving to session